consensus/reactor.go

```diff
@@ -595,7 +603,9 @@ OUTER_LOOP:
 						Part:   *parts,
 					},
 				}, logger) {
-					schema.WriteBlockPart(conR.traceClient, rs.Height, rs.Round, peer.ID(), part.Index, schema.TransferTypeUpload)
+					schema.WriteBlockPart(conR.traceClient, rs.Height,
+						rs.Round, peer.ID(), part.Index,
+						schema.TransferTypeUpload, int64(parts.Size()))
```

Comment:
To the reviewers: While an alternative approach could involve utilizing a helper function to measure the part size, it's worth noting that the `Size()` method within the protobuf message type `Part` already accomplishes the same task.